src/utils.ts

```diff
@@ -12,17 +13,30 @@ export namespace Utils {
         const esc = encodeURIComponent
         const headers = { 'authorization': payload['CF_API_KEY']! }
         const runtimeName = payload['CF_RUNTIME_NAME']
+        const platformHost = payload['CF_PLATFORM_URL']
+        const runtimeVersion = await Utils.getRuntimeVersion(headers, runtimeName, platformHost)
```

Comment:
If `runtime` will be empty, it will return empty string.
I think it may be better to call it only when runtime name was provided. This way we reduce the API call to platform from 2 to 1 and make this func to always return right semver number.
```js
		let shouldStorePayloadInHeaders = false
        if (!runtimeName) {
            host = payload['CF_HOST']
            delete payload['CF_HOST']
        } else {
            const runtimeEntity = await Utils.getRuntimeByName(runtimeName, headers, platformHost)
			host = runtimeEntity.host
			shouldStorePayloadInHeaders = gte(runtimeEntity.version, '0.0.553')
            delete payload['CF_RUNTIME_NAME']
            delete payload['CF_PLATFORM_URL']
        }
```